tic_tac_toe/game.py

```diff
@@ -24,16 +26,26 @@ def _position_is_valid(position):
 
     Returns True if given position is valid, False otherwise.
     """
-    pass
-
+    if (not isinstance(position, tuple) 
+        or not (isinstance(position[0], int) and isinstance(position[1], int))
+        or not len(position) == 2
+        or not ((0 <= position[0] < 3) and (0 <= position[1] < 3))):
+            
```

Comment:
Ahhh that's right there are numbers between 1 and 2. They totally could enter those values, you're correct. My bad I didn't think about floats.